test/e2e/cases/basic.go

```diff
@@ -16,13 +16,17 @@ package e2e
 
 import (
 	"fmt"
+	"strings"
 	"time"
 
 	"github.com/golang/glog"
 
 	"github.com/GoogleCloudPlatform/k8s-multicluster-ingress/app/kubemci/pkg/kubeutils"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
We really need to find out why I dont require this and you do :)
Is it just me who doesnt require it or is it just you who requires this :)